<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebShow</class>
 <widget class="QWidget" name="WebShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreeCharts</string>
  </property>
  <widget class="QFrame" name="frame_web">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>700</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ChartsStyles">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_X">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>110</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>81</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X轴数据：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>151</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Y轴数据：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Y">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>标题：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Title">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>270</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数据名(选填)：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DataName">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>330</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Generate">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
